### ISSUE TITLE
Erroneous UNASSIGNED-BestPractices-vkCreateDevice-physical-device-features-not-retrieved

### ISSUE THREAD

**Issue by timvpGoogle** [KhronosGroup/Vulkan-ValidationLayers#2216]
After updating the validation layers within ANGLE, we are seeing the following validation layer warning now:

```
WARN: RendererVk.cpp:340 (DebugUtilsMessenger): [ UNASSIGNED-BestPractices-vkCreateDevice-physical-device-features-not-retrieved ] Validation Warning: [ UNASSIGNED-BestPractices-vkCreateDevice-physical-device-features-not-retrieved ] Object 0: VK_NULL_HANDLE, type = VK_OBJECT_TYPE_DEVICE; | MessageID = 0x4387cf33 | vkCreateDevice() called before getting physical device features from vkGetPhysicalDeviceFeatures().
```

The code that the VVL is complaining about:

https://source.chromium.org/chromium/chromium/src/+/master:third_party/angle/src/libANGLE/renderer/vulkan/RendererVk.cpp;drc=9f5eb0b8fb29b35b89858c36272db16d9b7d2302;l=1427
```
    ANGLE_VK_TRY(displayVk, vkCreateDevice(mPhysicalDevice, &createInfo, nullptr, &mDevice));
```

However, `vkGetPhysicalDeviceFeatures()` is called beforehand:

https://source.chromium.org/chromium/chromium/src/+/master:third_party/angle/src/libANGLE/renderer/vulkan/RendererVk.cpp;drc=9f5eb0b8fb29b35b89858c36272db16d9b7d2302;l=840
```
    vkGetPhysicalDeviceFeatures(mPhysicalDevice, &mPhysicalDeviceFeatures);
```

It seems the VVL is forgetting about the call to `vkGetPhysicalDeviceFeatures()` now.

**Comment by ncesario-lunarg** [comment]
Fixed in #2220.
